tests/e2e/standalone/list_test.go

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation and Dapr Contributors.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListRunWhenNoDaprrd(t *testing.T) {
```

Comment:
It might make sense to group this test here: https://github.com/dapr/cli/blob/58263ce/tests/e2e/standalone/standalone_test.go#L394

Reply:
Thanks Charlie, latest change has the test moved to standalone_test.go and I have deleted the old test